deepchem/rl/torch_rl/torch_a2c.py

```diff
@@ -0,0 +1,129 @@
+"""Advantage Actor-Critic (A2C) algorithm for reinforcement learning."""
+import numpy as np
+from typing import List
+try:
+    import torch
+    has_pytorch = True
+except:
+    has_pytorch = False
+
+
+class A2CLossDiscrete(object):
```

Comment:
Do the losses inherit from object in general? Check the other deepchem losses

Reply:
The tensorflow implementation of these losses does inherit from object.